### PR TITLE
ADD: New frameworks and executor to the community list.

### PR DESCRIPTION
**Pull request by andreaspeters**
Some new Frameworks and Executors for the website.

### CODE REVIEW

File: docs/frameworks.md [hunk 1]
```diff
@@ -11,12 +11,15 @@ layout: documentation
 ## Long Running Services
 
 * [Aurora](http://aurora.apache.org) is a service scheduler that runs on top of Mesos, enabling you to run long-running services that take advantage of Mesos' scalability, fault-tolerance, and resource isolation.
+* [Compose](https://github.com/AVENTER-UG/mesos-compose) is a scheduler (HTTP API) for long running Mesos tasks. The syntax is compatible to docker-compose.
 * [Marathon](https://github.com/mesosphere/marathon) is a private PaaS built on Mesos. It automatically handles hardware or software failures and ensures that an app is "always on".
+* [M3s](https://github.com/AVENTER-UG/mesos-m3s) is a scheduler (HTTP API) to run multiple K3s (Kubernetes) cluster on top of Mesos.
```
Comment (qianzhangxa) [inline docs/frameworks.md:16]: I think this line should be put above the `Marathon` line.

File: docs/frameworks.md [hunk 2]
```diff
@@ -49,3 +52,13 @@ layout: documentation
 ## Machine Learning
 
 * [TFMesos](https://github.com/douban/tfmesos) is a lightweight framework to help running distributed [Tensorflow](https://www.tensorflow.org/) Machine Learning tasks on Apache Mesos with GPU support.
+
+## Load Balancing
+
+* [Traefik Mesos provider](https://github.com/AVENTER-UG/traefik-mesos) is a modern HTTP reverse proxy and load balancer with build in TCP and UDP support.
```
Comment (qianzhangxa) [inline docs/frameworks.md:58]: s/build in/built-in/

File: docs/frameworks.md [hunk 3]
```diff
@@ -49,3 +52,13 @@ layout: documentation
...
+
+## Executor
+
```
Comment (qianzhangxa) [inline docs/frameworks.md:61]: I do not think we should put it here since this doc is `frameworks.md`.

Comment (andreaspeters) [reply docs/frameworks.md:61]: :thinking: ok, maybe we should have a place where we can add community executors? :man_shrugging:

**Comment by qianzhangxa** [comment]
Thanks @andreaspeters for the contribution!

**Pull request merged**